dolphin/coordination.py

```diff
@@ -205,3 +218,24 @@ def _synchronized(f, *a, **k):
             return f(*a, **k)
 
     return _synchronized
+
+
+def get_redis_backend_url():
```

Comment:
plze use _ as first letter if it's internal function

Reply:
Done